src/basics.php

```diff
@@ -155,3 +155,16 @@ function loadPHPUnitAliases()
         require_once dirname(__DIR__) . DS . 'tests' . DS . 'phpunit_aliases.php';
     }
 }
+
+if (!function_exists('getVarType')) {
```

Comment:
If we intend to use this in classes that end up in sub split packages,this function should be in src/Core/functions.php instead.